beetsplug/info.py

```diff
@@ -52,7 +53,11 @@ def emitter():
             tags[field] = getattr(mf, field)
         tags['art'] = mf.art is not None
         tags['path'] = displayable_path(path)
+        # create a temporary Item to take advantage of __format__
+        tags['item'] = Item(db=None, **tags)
```

Comment:
Hmm; that's a very good point. It makes sense to keep the output for files as faithful as possible to the files themselves.

On further reflection, I think I like your current solution the best. Both emitters can produce the same kind of data—a pair of a dictionary (for the default mode) and an Item (for the --format mode). That way, the data output piece can stay agnostic with respect to the data source. Sorry for getting in your way when you had it right all along! :smiley: